src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let copyState = { ...state };
```

Comment:
The variable `copyState` is not necessary here. You can directly modify and use `newState` in each iteration of your `actions` loop to apply the changes.